crates/fuel-core/src/graphql_api/worker_service.rs

```diff
@@ -381,6 +386,64 @@ where
             )
             .into());
         }
+
+        for receipt in result.receipts() {
+            match receipt {
+                Receipt::Mint {
+                    sub_id,
+                    contract_id,
+                    val,
+                    ..
+                } => {
+                    let asset_id = contract_id.asset_id(sub_id);
+                    let current_count = match db.storage::<AssetsInfo>().get(&asset_id) {
```

Comment:
What we're getting back here is actually not a count. Can you consider renaming the binding?